src/data/markdown/docs/30 xk6-browser/01 xk6-browser/03 BrowserType.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "BrowserType"
+excerpt: "xk6-browser: BrowserType Class"
+---
+
+The `BrowserType` is the entry point into launching a browser process; `chromium` is currently the only supported `BrowserType`. To use it, import `chromium` from the top level module `/k6/x/browser`.
```

Comment:
```suggestion
The `BrowserType` is the entry point into launching a browser process; `chromium` is currently the only supported `BrowserType`. To use it, import `chromium` from the top level module `k6/x/browser`.
```

Reply:
Hey, I still see `/k6/x/browser` on b01db6f2.

Reply:
Hmm, it looks like i added it back in 🤔, i've removed it now in cbdbfaa2b28fffbab81284e72b58b93df165c449.